modules/ossm-jaeger-supported-configurations.adoc

```diff
@@ -0,0 +1,45 @@
+////
+This REFERENCE module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-supported-configuration_{context}"]
+= Jaeger supported configuration
+
+For the {ProductVersion} release of {ProductName}the only supported configuration of Jaeger is as follows:
+
+.Supported Jaeger configurations
+[options="header"]
+[cols="l, a"]
+|====
+|Spec|Notes
+|strategy: production
+storage:
+    type: elasticsearch
+
```

Comment:
Would it be worth including a customized version of the elasticsearch config, as in https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/examples/openshift/simple-prod-deploy-es.yaml ? Indicating that it is optional - and in that case the es-operator will use default values.

Reply:
Also should include the index cleaner config: https://www.jaegertracing.io/docs/1.13/operator/#elasticsearch-index-cleaner-job